Oh build fails
/lgtm cancel

```
       > internal/pkg/daemon/metrics/grpc.go:112:5: not enough arguments in call to m.IncSelinuxProfileAudit
       >   have (string, string, string, string, string, string)
       >   want (string, string, string, string, string, string, string)
       > make: *** [Makefile:166: build/security-profiles-operator] Error 1
       For full logs, run:
```

--- a/internal/pkg/daemon/metrics/metrics.go
+++ b/internal/pkg/daemon/metrics/metrics.go
@@ -306,7 +306,7 @@ func (m *Metrics) IncSelinuxProfileDelete() {
 // IncSelinuxProfileAudit increments the selinux profile audit counter for the
 // provided labels.
 func (m *Metrics) IncSelinuxProfileAudit(
-	node, namespace, pod, container, executable, scontext, tcontext string,
+	node, namespace, pod, container, scontext, tcontext string,
 ) {
 	m.metricSelinuxProfileAudit.WithLabelValues(
 		node, namespace, pod, container, scontext, tcontext,

--- a/test/tc_log_enricher_test.go
+++ b/test/tc_log_enricher_test.go
@@ -137,7 +137,6 @@ spec:
 		metrics := e.runAndRetryPodCMD(curlSpodCMD)
 		e.Regexp(fmt.Sprintf(`(?m)security_profiles_operator_seccomp_profile_audit_total{`+
 			`container="%s",`+
-			`executable="/usr/sbin/nginx",`+
 			`namespace="%s",`+
 			`node=".*",`+
 			`pod="%s",`+

